scripts/set_extension_name.py

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/python3
 
-import sys, os
+import sys, os, shutil
 from pathlib import Path
 
-if (len(sys.argv) != 2):
-    raise Exception('usage: python3 set_extension_name.py <name_for_extension>')
+shutil.copyfile(f'docs/NEXT_README.md', f'README.md')
```

Comment:
Should this be a move?

Reply:
My idea was that copy makes for it to be doable again (say you want to change name a second time). Then some other stuff (like test file) will break, but I though was a first step in making it less state-dependent.